dannce/interface.py

```diff
@@ -1132,8 +1118,7 @@ def dannce_train(params: Dict):
                 params["new_n_channels_out"],
                 params["dannce_finetune_weights"],
                 params["n_layers_locked"],
-                False,
-                True,
+                params["norm_method"],
```

Comment:
```suggestion
                norm_method=params["norm_method"],
```

@data-hound look good?

Reply:
I don't think this needs to be changed. Since, False and True were being passed as positional arguments, and then they were replaced with norm_method. But, it is good to have this passed with keywords @spoonsso